front/lib/resources/file_resource.ts

```diff
@@ -83,6 +88,10 @@ export class FileResource extends BaseResource<FileModel> {
     });
   }
 
+  static baseFileUrl(owner: LightWorkspaceType): string {
```

Comment:
Based on my previous comment this might be deadcode

Reply:
This is temporary, we will rework this API to accept either a fileId or a content. This is an intermediary state.